### ISSUE TITLE
使用单个gpu训练，应该怎么修改defrcn/modeling/roi_heads/roi_heads.py中的代码？

### ISSUE THREAD

**Issue by lixi92** [WuShuang1998/MFDC#12]
有一些使用到分布式的代码，例如这段代码：
   `def concat_all_gather(tensor):
    tensors_gather = [
        torch.ones_like(tensor)
        for _ in range(torch.distributed.get_world_size())
    ]
    torch.distributed.all_gather(tensors_gather, tensor, async_op=False)
    output = torch.cat(tensors_gather, dim=0)
    return output`
应该如何修改？

**Comment by initialqing** [comment]
我也有相同的问题 请问您解决了吗？

**Comment by lixi92** [comment]
> 我也有相同的问题 请问您解决了吗？

没有呢，目前是改为在多个GPU上训练

**Comment by longwenwu** [comment]
> > 我也有相同的问题 请问您解决了吗？
> 
> 没有呢，目前是改为在多个GPU上训练

请问您是否解决了单gpu训练的问题呢？

**Comment by lixi92** [comment]
> > > 我也有相同的问题 请问您解决了吗？
> > 
> > 
> > 没有呢，目前是改为在多个GPU上训练
> 
> 请问您是否解决了单gpu训练的问题呢？

没有呢

**Comment by longwenwu** [comment]
> > > > 我也有相同的问题 请问您解决了吗？
> > > 
> > > 
> > > 没有呢，目前是改为在多个GPU上训练
> > 
> > 
> > 请问您是否解决了单gpu训练的问题呢？
> 
> 没有呢

您好 是不是文中提出的方法得用多gpu分布训练才行呢？目前我就是fine-tunning阶段运行不了
